api/src/main/java/io/strimzi/api/kafka/model/connect/build/ImageStreamOutput.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.connect.build;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.Constants;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Represents Docker output from the build
+ */
+@Buildable(
+        editableEnabled = false,
+        builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "image" })
+@EqualsAndHashCode
+public class ImageStreamOutput extends Output {
+    private static final long serialVersionUID = 1L;
+
+    private String image;
```

Comment:
```suggestion
    private String imageStream;
```
Shouldn't this technically be imageStream, not image?

Reply:
I and change it if you want. But I think that in terms of meaning it is close and makes the API slightly better to use the same for both `type: docker` and `type: imagestream`. So I would probably keep it as it is.